zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/builder/component/CheckBoxBuilder.kt

```diff
@@ -44,6 +57,6 @@ class CheckBoxBuilder(
     companion object {
 
         @JvmStatic
-        fun newBuilder() = CheckBoxBuilder()
+        fun newBuilder() = CheckBoxBuilder().apply { contentSize = Size.create(DefaultCheckBoxRenderer.BUTTON_WIDTH,1) }
```

Comment:
I realise it might be better to set the contentSize in an init block rather than here because the Size(1,1) bug is still present if someone is instanciating a CheckBoxBuilder directly

Reply:
Changed it